src/components/atoms/Button/index.tsx

```diff
@@ -0,0 +1,21 @@
+import clsx from 'clsx';
+import { ComponentProps } from 'react';
+
+interface Props {
+  type: ComponentProps<'button'>['type'];
+  onClick: ComponentProps<'button'>['onClick'];
+}
+
+export const Button = ({
+  type,
+  onClick,
+  children,
+  className,
+  ...buttonProps
+}: Props & ComponentProps<'button'>) => {
+  return (
+    <button className={clsx('btn-init', className)} type={type} onClick={onClick} {...buttonProps}>
+      {children}
+    </button>
```

Comment:
`type={type} onClick={onClick}`도 `{...buttonProps}`로 처리될 거 같아요. 제거해보면 어떨까요?

```suggestion
    <button className={clsx('btn-init', className)} {...buttonProps}>
      {children}
    </button>
```

Reply:
오 그러네요. 간소화할 수 있는 부분 알려주셔서 감사합니다 👍